### PR TITLE
wip: mainnet deploy configs

### PR DESCRIPTION
**Pull request by prestwich**

### CODE REVIEW

**Comment by prestwich** [comment]
need to replace the governor address with msig

File: typescript/optics-deploy/config/mainnets/celo.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  rpc,
+  deployerKey: process.env.CELO_DEPLOYER_KEY,
+  domain: 0x63656c6f, // b'eth' interpreted as an int
+};
+
+export const chain = toChain(chainJson);
+
+export const config: CoreConfig = {
+  environment: 'prod',
+  updater: '0xDB2091535eb0Ee447Ce170DDC25204FEA822dd81',
```
Comment (prestwich) [inline typescript/optics-deploy/config/mainnets/celo.ts:24]: @yourbuddyconner verify

File: typescript/optics-deploy/config/mainnets/celo.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  recoveryTimelock: 60 * 60 * 24, // 1 day
+  recoveryManager: '0x3D9330014952Bf0A3863FEB7a657bfFA5C9D40B9',
```
Comment (prestwich) [inline typescript/optics-deploy/config/mainnets/celo.ts:26]: @barbaraliau verify

File: typescript/optics-deploy/config/mainnets/celo.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  optimisticSeconds: 60 * 60 * 3, // 3 hours
+  watchers: ['0xeE42B7757798cf495CDaA8eDb0CC237F07c60C81'],
```
Comment (prestwich) [inline typescript/optics-deploy/config/mainnets/celo.ts:28]: @yourbuddyconner verify

File: typescript/optics-deploy/config/mainnets/ethereum.ts [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+export const config: CoreConfig = {
+  environment: 'prod',
+  updater: '0xDB2091535eb0Ee447Ce170DDC25204FEA822dd81',
+  recoveryTimelock: 60 * 60 * 24, // 1 day
+  recoveryManager: '0x3D9330014952Bf0A3863FEB7a657bfFA5C9D40B9',
+  optimisticSeconds: 60 * 60 * 3, // 3 hours
+  watchers: ['0xeE42B7757798cf495CDaA8eDb0CC237F07c60C81'],
+  governor: {
+    domain: chainJson.domain,
+    address: '0x5Fa96B622D1F4e920b92040c10fA297ca496ad37',
```
Comment (prestwich) [inline typescript/optics-deploy/config/mainnets/ethereum.ts:31]: @barbaraliau verify

File: typescript/optics-deploy/config/mainnets/celo.ts (hunk 1 above)
Comment (yourbuddyconner) [reply typescript/optics-deploy/config/mainnets/celo.ts:24]: Verified, correct

File: typescript/optics-deploy/config/mainnets/celo.ts (hunk 3 above)
Comment (yourbuddyconner) [reply typescript/optics-deploy/config/mainnets/celo.ts:28]: Verified, correct

**Review by yourbuddyconner — APPROVED** [review]
Watcher and Updater attest keys confirmed

File: typescript/optics-deploy/config/mainnets/ethereum.ts (hunk 4 above)
Comment (barbaraliau) [reply typescript/optics-deploy/config/mainnets/ethereum.ts:31]: verified

File: typescript/optics-deploy/config/mainnets/polygon.ts [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  domain: 0x706f6c79, // b'poly' interpreted as an int
+};
+
+export const chain = toChain(chainJson);
+
+export const config: CoreConfig = {
+  environment: 'prod',
+  updater: '0xDB2091535eb0Ee447Ce170DDC25204FEA822dd81',
+  recoveryTimelock: 60 * 60 * 24, // 1 day
+  recoveryManager: '0x3D9330014952Bf0A3863FEB7a657bfFA5C9D40B9',
```
Comment (barbaraliau) [inline typescript/optics-deploy/config/mainnets/polygon.ts:26]: verified

File: typescript/optics-deploy/config/mainnets/ethereum.ts [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  domain: 0x657468, // b'eth' interpreted as an int
+};
+
+export const chain = toChain(chainJson);
+
+export const config: CoreConfig = {
+  environment: 'prod',
+  updater: '0xDB2091535eb0Ee447Ce170DDC25204FEA822dd81',
+  recoveryTimelock: 60 * 60 * 24, // 1 day
+  recoveryManager: '0x3D9330014952Bf0A3863FEB7a657bfFA5C9D40B9',
```
Comment (barbaraliau) [inline typescript/optics-deploy/config/mainnets/ethereum.ts:26]: verified

File: typescript/optics-deploy/config/mainnets/celo.ts (hunk 2 above)
Comment (barbaraliau) [reply typescript/optics-deploy/config/mainnets/celo.ts:26]: verified

**Review by barbaraliau — APPROVED** [review]
Verified `recoveryManager` and `governor` addresses

**Pull request merged**